CHANGELOG.md

```diff
@@ -10,6 +10,11 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## [Unreleased]
 
+- 🎁 A new option `--log-directory` has been added, which allows storing the log
+  separate from the persistent state. If unset, this option falls back to using
+  the persistent state directory set using `--directory`.
```

Comment:
Technically, a log is also persistent, so the differentiation could be a bit clearer. Here's an alternate suggestion:

>  The new option `--log-directory` allows for storing the VAST application logs separate from the database state. If unset, this option falls back to  `--directory`.